### DISCUSSION TITLE
Request of New Name

### DISCUSSION THREAD

**Discussion by kakkun61** [kakkun61/ghcman#16] [Q&A] [answered]
ghcups is confusing some users because the name of ghcups is similar to ghcup.

I have associated ghcups with “ghcup + ps”, ps is an abbrev of PowerShell.

Would you tell me your ideas for a new name for this tool?

**Comment by jneira** [comment]
Hi, thanks for this handy tool. I am not using it yet but i am considering switch from my manual haskell setup.

What about add "win" as part of the name? It would make clear that is a tool targeting windows systems, maybe ghcup-win, win-ghcup, ghcwp 😉 ?

**Comment by kakkun61** [comment]
ysangkok's ideas

- ghcup-powershell
- win-ghcup

https://www.reddit.com/r/haskell/comments/e4ypkb/announce_ghcups_is_released/f9lhhcx?context=3

**Reply by kakkun61** [reply]
This tool may support Linux and/or Mac OS in the future for users of PowerShell Core on other OSes. And so I don't want to add “win” or something to its name.

**Comment by kakkun61** [comment, marked as answer]
Current my idea is

- ghcman.

This is an abbrev of GHC Manager.
